codespell_lib/data/dictionary.txt

```diff
@@ -6871,7 +6880,7 @@ entired->entered, entire,
 entireity->entirety
 entires->entries
 entirey->entirely
-entirity->entirely
+entirity->entirety, entirely,
```

Comment:
I'd agree with your suggestion and be tempted to drop entirely, it seems like it was influenced by the ones either side of it.

Reply:
I'll drop it